src/lib/core/fiber_channel.h

```diff
@@ -351,15 +362,14 @@ fiber_channel_count(struct fiber_channel *ch)
 }
 
 /**
- * Close the channel. Discards all messages
- * and wakes up all readers and writers.
+ * Close the channel for writing. If not graceful, discards all
+ * messages and wakes up all readers and writers.
```

Comment:
'Is graceful' here without a context may be confusing. I would at least link a setter for the flag.

Reply:
```
  /**
   * Close the channel for writing. If not `fiber_channel_graceful_close`, 
   * discards all messages and wakes up all readers and writers.                 
   */                                                                            
```